src/transformers/modeling_tf_openai.py

```diff
@@ -199,7 +199,7 @@ def call(self, inputs, training=False):
 
 class TFOpenAIGPTMainLayer(tf.keras.layers.Layer):
     def __init__(self, config, *inputs, **kwargs):
-        super().__init__(config, *inputs, **kwargs)
+        super().__init__(*inputs, **kwargs)
```

Comment:
Bugfix, independent of this PR and posted separately in #3115, but included here because it got in the way of testing.